todos/app/resources/projects/resource.rb

```diff
@@ -2,6 +2,8 @@
 
 module Projects
   class Resource < ApplicationResource
+    filter_by :page, optional: true
```

Comment:
I think this should be a different method called `paginate`, where you set the (optional) options for Kaminari. I also think the default should be to paginate: you would only use the `paginate` method if you want to customize Kaminari. What do you think?

Reply:
Agreed, I just didn't find where to set this default :page attribute. Inside Filters?

Reply:
We also need to accept a per_page param.

Reply:
This was a hack to have the easiest way to permit that param on json schema.